packages/mdc-top-app-bar/adapter.js

```diff
@@ -47,6 +47,19 @@ class MDCTopAppBarAdapter {
    */
   hasClass(className) {}
 
+  /**
+   * Adds the specified attribute and value to the root Element.
+   * @param {string} attribute
+   * @param {string} value
+   */
+  setStyle(attribute, value) {}
```

Comment:
Nit: Rename `attribute` to `property`

(See https://css-tricks.com/css-ruleset-terminology/)